src/cljs/main/broadfcui/page/profile.cljs

```diff
@@ -36,7 +44,13 @@
            expiring-soon? (< expire-time (utils/_24-hours-from-now-ms))
            datasets (:datasetPermissions status)]
        [:div {}
-        [:h3 {} "Linked NIH Account"]
+        [:div {}
+         [:h4 {:style {:display "inline-block"}} "NIH Account"]
```

Comment:
Instead of wrapping this in a `div`, try putting the info-box inside the `h4` and removing the `:display`